Create new lightweight image for cluster operations (#795)
kyma-project/test-infra: prow/images/alpine-kubectl/README.md

# Kubectl Docker Image

## Overview

> This folder contains a minimal docker image that might be used for cluster control. 

Reviewer: ```suggestion
This folder contains a minimal Docker image that you can use to control the cluster.
```
